client/src/components/Lists.jsx

```diff
@@ -26,12 +28,16 @@ class Lists extends Component {
 	}
 
 	renderLists() {
-		const lists = [];
+    const lists = [];
 		this.props.lists.map((list) => {
-			let { title, id } = list;
+      let { title, id } = list;
+      // TODO: return directly, instead of mapping.
```

Comment:
i.e.

```tsx
this.props.list((list) => (
  <div className="listItem"
...
</div>
)}
```

Reply:
May need quick clarification on this for next session